qiskit_experiments/database_service/db_analysis_result.py

```diff
@@ -158,9 +158,9 @@ def save(self) -> None:
         value = self.value
         result_data = {
```

Comment:
Please write an internal comment here explaining why we have  for this dict both the `_value` and `_chisq` key pairs as well as `value` and `chisq` which are added later on.